zellij-server/src/panes/tiled_panes/mod.rs

```diff
@@ -378,7 +379,8 @@ impl TiledPanes {
     pub fn has_panes(&self) -> bool {
         !self.panes.is_empty()
     }
-    pub fn render(&mut self, output: &mut Output, floating_panes_are_visible: bool) {
+    pub fn render(&mut self, output: &mut Output, floating_panes_are_visible: bool) -> Result<()> {
+        let err_context = || "failed to render output based on visible";
```

Comment:
I get your intention here, but I don't think we should mention the "visible" condition in the error. What about: "failed to render tiled panes"?

(Same goes for the err_context further down)